intro/script/answers.py

```diff
@@ -1,12 +1,17 @@
 import pandas as pd
 import matplotlib.pyplot as plt
-
+import json
+from pprint import pprint
 plt.style.use('ggplot')
 plt.rcParams['font.family'] = 'sans-serif'
 
 
 def operating_system(answers):
-    os = answers['Betriebssystem'].value_counts()
+    liste = []
+    for participant in answers:
+        if (participant['toolbox'] == True):
```

Comment:
python does not use parenthese around if conditions and comparisons to True should always be `is True` not `== True`.